pkg/agent/querier/querier.go

```diff
@@ -97,6 +98,11 @@ func NewAgentQuerier(
 	}
 }
 
+// GetFQDNCache returns dnsEntryCache within fqdnController
+func (aq agentQuerier) GetFqdnCache(fqdnFilter *querier.FQDNCacheFilter) []types.DnsCacheEntry {
+	return aq.networkPolicyInfoQuerier.GetFQDNCache(fqdnFilter)
+}
```

Comment:
This function should not be needed and I actually don't think it's used anywhere?